### DISCUSSION TITLE
Ad blocking Log message

### DISCUSSION THREAD

**Discussion by christiano-git** [RaspAP/raspap-webgui#1182] [Q&A]
Dear all,
I started to use RaspAP and tested the Ad Blocker. Below i copied 2 Log example.
I am wondering what "config" means. Is this a other file or ?

1) is blocked as per the hostname.txt
2) is mention "config"

Thanks for feedback

1) xxxx dnsmasq [7240]: /etc/raspap/adblock/hostnames.txt xxxx.avast.com is 0.0.0.0
2) xxxx dnsmasq [9019]: config xxxxx.urbanairship.com is 0.0.0.0

**Comment by billz** [comment]
RaspAP's ad block logging reads the dnsmasq log, by default located at `/var/log/dnsmasq.log`, and searches for any line with a "0.0.0.0" pattern. The ad block [list source](https://docs.raspap.com/adblock/#blocklist-source) used by RaspAP maps undesirable hosts to this 0.0.0.0 IPv4 address.

The "config" filename probably references the default `/etc/dnsmasq.conf`. RaspAP does not touch this file, but it may have been modified by something else. Check the contents of the default config and the files located in `/etc/dnsmasq.d/`.
